rpc/client/local/app_test.go

```diff
@@ -0,0 +1,45 @@
+package local_test
+
+import (
+	"math/rand"
+
+	meapp "github.com/tendermint/merkleeyes/app"
+
+	wire "github.com/tendermint/go-wire"
+)
+
+// MakeTxKV returns a text transaction, allong with expected key, value pair
+func MakeTxKV() ([]byte, []byte, []byte) {
```

Comment:
No, I meant that we should probably move this into one package and reuse rather than duplicating the code.